modules/interactive_music/doc_classes/AudioStreamPlaybackInteractive.xml

```diff
@@ -9,6 +9,19 @@
 	<tutorials>
 	</tutorials>
 	<methods>
+		<method name="get_current_clip_index">
+			<return type="int" />
+			<description>
+				Return the index of the currently playing clip.
+				You can use this to get the name of the currently playing clip with [method AudioStreamInteractive.get_clip_name].
+				[b]Example:[/b] Getting the currently playing clip name from inside an AudioStreamPlayer node.
```

Comment:
```suggestion
				[b]Example:[/b] Get the currently playing clip name from inside an [AudioStreamPlayer] node.
```

Reply:
Changes made. Sorry for the grammar mistakes.